acoustid/data/fingerprint.py

```
@@ -2,10 +2,11 @@
 # Distributed under the MIT license, see the LICENSE file for details.
 
 import logging
-from typing import Any, Dict, List, NamedTuple, Optional
+from typing import Any, Dict, List, NamedTuple, Optional, cast
 
-from sqlalchemy import sql
+from sqlalchemy import func, literal_column, select, sql, text
 from sqlalchemy.exc import OperationalError
+from sqlalchemy.sql.elements import BooleanClauseList, ColumnElement
```

Ruff (0.8.2)

`sqlalchemy.sql.elements.BooleanClauseList` imported but unused

Remove unused import: `sqlalchemy.sql.elements.BooleanClauseList`

(F401)